lib/pact/matchers/matchers.rb

```diff
@@ -39,6 +40,7 @@ def calculate_diff expected, actual, opts = {}
       when Regexp then regexp_diff(expected, actual, options)
       when Pact::SomethingLike then calculate_diff(expected.contents, actual, options.merge(:type => true))
       when Pact::ArrayLike then array_like_diff(expected, actual, options)
+      when Pact::Jwt then jwt_diff(expected,actual,options)
```

Comment:
Ok, I know it's just a conversation stater, but it's nice to make a good first impression ;)